### PR TITLE
Aedb 114 acceptance test employer create demand unverified

### PR DESCRIPTION
**Pull request by A-J-Wright**

### CODE REVIEW

File: src/SFA.DAS.EmployerDemand.Web.AcceptanceTests/Steps/HttpSteps.cs [hunk 1]
```diff
@@ -57,7 +58,35 @@ public void ThenAnHttpStatusCodeIsReturned(int httpStatusCode)
...
+        public async Task WhenIPostTheViewedPageAsync()
+        {
+            if (!_context.TryGetValue<HttpResponseMessage>(ContextKeys.HttpResponse, out var result))
+            {
+                Assert.Fail($"scenario context does not contain value for key [{ContextKeys.HttpResponse}]");
+            }
+
+            var client = _context.Get<HttpClient>(ContextKeys.HttpClient);
+            HttpContent content = null;
+            var continuedResponse = await client.PostAsync(result.RequestMessage.RequestUri.PathAndQuery, content);
```
Comment (dashton82) [inline src/SFA.DAS.EmployerDemand.Web.AcceptanceTests/Steps/HttpSteps.cs:85]: Can remove the `HttpContent content = null;` and just pass null in the PostAsync

```suggestion
            var continuedResponse = await client.PostAsync(result.RequestMessage.RequestUri.PathAndQuery, null);

```

File: src/SFA.DAS.EmployerDemand.Web.AcceptanceTests/Features/EmployerCreateDemandUnverified.feature [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿Feature: EmployerCreateDemandUnverified
+	As ESFA
+	I want to tell the employer to verify their email
+	so that I know their contact details are correct
+
+@WireMockServer
+	Scenario: Valid Employer Demand Submittied
```
Comment (dashton82) [inline src/SFA.DAS.EmployerDemand.Web.AcceptanceTests/Features/EmployerCreateDemandUnverified.feature:7]: This scenario can be moved into this feature file https://github.com/SkillsFundingAgency/das-employerdemand-web/blob/main/src/SFA.DAS.EmployerDemand.Web.AcceptanceTests/Features/EmployerDemandForm.feature

**Review by dashton82 — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.EmployerDemand.Web.AcceptanceTests/Features/EmployerCreateDemandUnverified.feature (hunk 2 above)
Comment (A-J-Wright) [reply src/SFA.DAS.EmployerDemand.Web.AcceptanceTests/Features/EmployerCreateDemandUnverified.feature]: Done

**Review by A-J-Wright — COMMENTED** [review]
changes made

**Review by dashton82 — APPROVED** [review]

**Pull request merged**